Code/GraphMol/DistGeomHelpers/Embedder.cpp

```diff
@@ -1050,6 +1051,49 @@ void embedHelper_(int threadId, int numThreads, EmbedArgs *eargs,
     delete positions[i];
   }
 }
+
+std::vector<std::vector<unsigned int>> getMolSelfMatches(
```

Comment:
I think it would be useful to make this a utility class, something like:

```
bool symmetry=true;
RMSDSet symm(mol, symmetry);
if (symm.rmsd(conf)) < 0.5) {
   symm.add(conf);
}
```

Perhaps not for this PR.